internal/pkg/template/diff/diff.go

```diff
@@ -159,11 +184,19 @@ func parseSequence(fromNode, toNode *yaml.Node) ([]diffNode, error) {
 		return nil, nil
 	}
 	var children []diffNode
+	var matchCount int
 	inspector := newLCSStateMachine(fromSeq, toSeq, lcsIndices)
 	for action := inspector.action(); action != actonDone; action = inspector.action() {
```

Comment:
:lolsob: there is a little typo here 
```suggestion
	for action := inspector.action(); action != actionDone; action = inspector.action() {
```